### DISCUSSION TITLE
"Color and fonts" preset for the high-contrast dark theme

### DISCUSSION THREAD

**Discussion by HubKing** [dbeaver/dbeaver#21894] [Q&A] [answered]
Available themes are: High contrast, dark, classic, light. My OS's theme is Gnome Dark, and choosing classic or light seemed to have problems like white text on light background. The "dark" theme lacked contrast which made it difficult to read. So, I chose the "high contrast".

The problem is that even this does not look correct. In the table view things like the null value or the [ ] look black on dark background. I could manually correct the colour for the null value, but there are so many items so to find out which one is which and editing them manually feel stupid. Also, I think UI elements need to have white border otherwise it is difficult to see where an element start.

In short, is there a "Colour and fonts" preset for high-contrast dark theme? It does not seem to have any "import" feature. How can I import it?

**Comment by E1izabeth** [comment, marked as answer]
There is no such preset. However, we are happy to improve our High-contrast theme. Could you create an issue about that and describe all places that are not well yet?

**Reply by HubKing** [reply]
Created https://github.com/dbeaver/dbeaver/issues/21895
